src/openrct2/network/Network.cpp

```diff
@@ -262,7 +262,7 @@ bool Network::BeginClient(const char* host, uint16 port)
         }
     } else {
         // LoadPrivate returns validity of loaded key
-        bool ok = false;
```

Comment:
I think I prefer the old notation. I know it will be false, but this is more explicit.